app/src/main/java/net/kwatts/powtools/drawables/TrackDrawable.kt

```diff
@@ -0,0 +1,68 @@
+package net.kwatts.powtools.drawables
+
+import android.graphics.*
+import android.graphics.drawable.Drawable
+import kotlin.math.absoluteValue
+
+class TrackDrawable @JvmOverloads constructor(
+        track: List<Pair<Double?, Double?>>,
+        private val scale: Float = 1f,
+        private val trackWidth: Float = 1f,
```

Comment:
would probably be beneficial to document how these numbers work. trackWidth isn't pixels, right?
or if you don't want to get technical at least document what numbers worked for you in the scenarios you tested. 
(ie for a track within 37.1234,-120.1234 to 37.1235,-120.1235 I used scale 1000 and trackWidth .2f and it looked pretty good)

Reply:
Removed `trackWidth` parameter in favor of a default stroke width that plays nice with the default scale.

`setTrackWidth(float)` method can be used to change width (parameter is in pixels).